src/main/resources/io/jenkins/blueocean/autofavorite/user/FavoritingUserProperty/config.properties

```diff
@@ -0,0 +1 @@
+autofavorite.enabled = when your email address appears in the change list of a run
```

Comment:
FFS this is where it had to go... (https://wiki.jenkins.io/display/JENKINS/Internationalization#Internationalization-Markingmessagesinjellyfiles didn't point that out)

Reply:
It took FOREVER.